### PR TITLE
Update faq.md

### PR DESCRIPTION
**Pull request by chickenrae**
Fixed this issue: doc: reword some FAQ answers #18

Signed-off-by: Signed-off-by: Rachel Roumeliotis <rroumeliotis@gmail.com>

### CODE REVIEW

File: faq.md [hunk 1]
```diff
@@ -24,22 +24,22 @@ Open Platform for Enterprise AI
...
+- MinIO 
+- Qdrant 
+- Red Hat 
+- SAS 
+- VMware by Broadcom 
+- Yellowbrick Data 
+- Zilliz 
 
 ## What is Intel contributing? 
 OPEA is to be defined jointly by several community partners, with a call for broad ecosystem contribution, under the well-established LF AI & Data Foundation. As a starting point, Intel has contributed a Technical Conceptual Framework that shows how to construct and optimize curated GenAI pipelines built for secure, turnkey enterprise deployment.  At launch, Intel contributed several reference implementations on Intel hardware across Intel® Xeon® 5, Intel® Xeon® 6 and Intel® Gaudi® 2, which you can see in a Github repo here. Over time we intend to add to that contribution including a software infrastructure stack to enable fully containerized AI workload deployments as well as potentially implementations of those containerized workloads.
```
Comment (mkbhanda) [inline faq.md:45]: which you can see in a Github repo **here**.  Here implies a link. Why not just say, available in the GitHub repo.  BTW, it is GitHub.

s/Over time we intend to add to that contribution including a software infrastructure stack to enable fully containerized AI workload deployments as well as potentially implementations of those containerized workloads./ Over time we shall include additional genAI pipelines, microcomponents, more cloud infrastructure support and round out evaluation with respect to safety, accuracy, scalability, and performance.

**Review by mkbhanda — COMMENTED** [review]
a few edits suggested.

File: faq.md [hunk 2]
```diff
@@ -24,22 +24,22 @@ Open Platform for Enterprise AI
...
-KX 
-MariaDB Foundation 
-MinIO 
-Qdrant 
-Red Hat 
-SAS 
-VMware by Broadcom 
-Yellowbrick Data 
-Zilliz 
+## Initial partner companies that joined OPEA in April 2024?
```
[the remark this answers is not in the document]
Comment (dbkinder) [reply faq.md:27]: You could remove this topic from the FAQ and instead just refer them to the https://opea.dev/ home page where you have a list of contributors (and their logos).  Do you want to keep this initial list of April companies or is that not worth remembering?

File: faq.md [hunk 3]
```diff
@@ -73,11 +73,11 @@ A version of the spec is available in the docs repo in this project
...
 There is no cost for anyone to join and contribute. 
 
-## Do I need to be Linux Foundation member to join?  
+## Do I need to be a Linux Foundation member to join?  
 Anyone can join and contribute. You don’t need to be a Linux Foundation member. 
 
-## Where can I report a bug?
-Vulnerability reports can be sent to info@opea.dev.
+## Where can I report a security vulnerability?
+Vulnerability reports can be sent to security@opea.dev.
```
Comment (dbkinder) [inline faq.md:80]:
```suggestion
Send vulnerability reports to security@opea.dev.
```
I suspect we'd want to include what information they should include in that report email?

**Review by dbkinder — CHANGES_REQUESTED** [review]
A few suggestions...

**Comment by dbkinder** [comment]
You need to add a Signed-off-by line to the commit message, something done automatically via the command line with "git commit -s".  If you're using the GitHub editor, you need to manually add a line saying something like this (but with your GitHub user name and email address.
```
Signed-off-by: Signed-off-by: David B. Kinder <david.b.kinder@intel.com>
```

File: faq.md (hunk 3 above)
Comment (mkbhanda) [reply faq.md:80]: Typically, we provide a template for all issues and perhaps a different one for security issues. We could follow the patterns on Kubernetes, or other project.

Comment (dbkinder) [reply faq.md:80]: But we don't want to create a GitHub issue for security concerns. Such reporting needs to be done privately.

File: faq.md (hunk 1 above)
Comment (chickenrae) [reply faq.md:45]: This doc lived pre-the github repo, we can change

**Review by dbkinder — APPROVED** [review]

**Comment by craigmateo** [comment]
updated with #89

**Pull request closed without merging**